lib/libc/stdio/printf.3

```diff
@@ -583,7 +603,26 @@ Zeroes are always represented with a mantissa of 0 (preceded by a
 .Ql -
 if appropriate) and an exponent of
 .Li +0 .
+.Pp
+.It Cm B
+The
+.Vt unsigned int
+(or apporiate variant) argument is converted to unsigned binary.
```

Comment:
This is a little thin compared to, say, `o` or `x`.

Reply:
i agree, would you recommend evaluating what o and x look like and see how it aligns with B since they are similar specifiers?

Reply:
Yes.